src/Modal/README.md

```diff
@@ -0,0 +1 @@
+# Modal
```

Comment:
Need a bit more here

Reply:
Should I focus on this or on the doc site md?

Reply:
This can be a follow-up from our chat. I think there will be several doc site tickets.